EchoPro/survey.py

```diff
@@ -315,10 +258,17 @@ def read_validated_data( self ,
             # If kriging dataset, then step one layer deeper into dictionary
             elif config_map[0] == 'kriging':
                 internal = internal['kriging']    
-            
+
             # A single dataframe per entry is expected, so no other fancy operations are needed
-            df_list = [internal[config_map[1] + '_df'] , df]
-            internal[config_map[1] + '_df'] = pd.concat(df_list)
+            # If geo_strata, differentiate between inpfc and stratification1 sheets
+            ### TODO: Temporary approach for incorporating the inpfc dataset. An improved approach
+            ### can be incorporated later on.
```

Comment:
Suggest adding this as an issue for tracking.

Reply:
This has been added as an issue (#178).